src/game/client/cdll_int.cpp

```diff
@@ -24,7 +24,11 @@ extern "C"
 }
 
 #include <cstring>
+// Another caveat of compiling for for i386 with a 10.13 SDK, no access to filesystem - they're very slow to adapt
+#ifndef PLATFORM_MACOS
 #include <filesystem>
```

Comment:
Filesystem is standard C++17 library. Why isn't it including in MacOS?

Reply:
<filesystem> header was added in toolchain for macOS 10.15 which doesn't support 32-bit apps. 10.13 should have <experimental/filesystem> but I'm not sure if it's stable.

Reply:
I've tried to tackle the same issue before when fixing up HLEnhanced for macOS. The only solution I came up with was just gutting it entirely. The latest Xcode available on High Sierra is 10.1 and it doesn't include any version of the `filesystem` headers. I was honestly thinking of converting the existing code to use the existing filesystem interface. Now I think a solution to this is maybe to enforce that Apple LLVM can't be used (unless it's a version shipped with 10.15, but then 32-bit support would be dropped), and something like an LLVM from Homebrew be used instead, as it seems this version doesn't shirk on support.